ui/js/reducers/claims.js

```diff
@@ -190,6 +197,55 @@ reducers[types.CREATE_CHANNEL_COMPLETED] = function(state, action) {
   });
 };
 
+reducers[types.SET_SUPPORT_AMOUNT] = function(state, action) {
```

Comment:
All of this can be simplified by having the component send these values directly to the action, unless I'm missing the need for these to be in Redux state.